openapi-cli/src/main/java/io/ballerina/generators/GeneratorUtils.java

```diff
@@ -308,6 +309,44 @@ public static String escapeIdentifier(String identifier) {
         return identifier;
     }
 
+    /**
+     * Generate operationId by removing special characters.
+     *
+     * @param identifier input function name, record name or operation Id
+     * @return string with new generated name
+     */
+    public static String getValidName(String identifier, boolean isSchemaName) {
+        if (!identifier.matches("\\b[a-zA-Z][a-zA-Z0-9]*\\b") && !identifier.matches("\\b[0-9]*\\b")) {
+            String[] split = identifier.split(GeneratorConstants.ESCAPE_PATTERN);
+            StringBuilder operationName = new StringBuilder();
```

Comment:
`operatoinName` must be `validName` right? We are not getting a valid name only for operation id, also for schema name as well.

Reply:
yeah, will change with `validName`

Reply:
resovled with [https://github.com/ballerina-platform/ballerina-openapi/pull/291/commits/0479016e743216e05b5282cfa2d4890bf19d33cb](https://github.com/ballerina-platform/ballerina-openapi/pull/291/commits/0479016e743216e05b5282cfa2d4890bf19d33cb)